common/curl-p/hamming.c

```diff
@@ -18,7 +19,8 @@ short test(PCurl *const curl, unsigned short const security) {
     sum = 0;
 
     for (j = 0; j < security; j++) {
-      for (k = j * HASH_LENGTH / 3; k < (j + 1) * HASH_LENGTH / 3; k++) {
+      for (k = j * HASH_LENGTH_TRIT / 3; k < (j + 1) * HASH_LENGTH_TRIT / 3;
```

Comment:
While we're at it, we should also replace the various `HASH_LENGTH_TRIT / 3` with `HASH_LENGTH_TRYTE`